Sources/MapboxCoreNavigation/Telemetry/NavigationCommonEventsManager.swift

```diff
@@ -187,57 +177,36 @@ open class NavigationEventsManager {
 
     func start() {
         let shortVersion = Bundle.navigationSDKVersion
-        eventsAPI.sendTurnstileEvent(sdkIdentifier: NavigationEventsManager.userAgent, sdkVersion: shortVersion)
+        eventsAPI.sendTurnstileEvent(sdkIdentifier: NavigationCommonEventsManager.userAgent, sdkVersion: shortVersion)
     }
     
     // MARK: Sending Feedback Events
-    
-    /**
-     Create feedback about the current road segment/maneuver to be sent to the Mapbox data team.
-     
-     You can pair this with a custom feedback UI in your app to flag problems during navigation such as road closures, incorrect instructions, etc.
-     
-     - returns: Returns a feedback event.
-     
-     If you provide a custom feedback UI that lets users elaborate on an issue, you should call this before you show the custom UI so the location and timestamp are more accurate.
-     Alternatively, you can use `FeedbackViewContoller` which handles feedback lifecycle internally.
-     
-     - Postcondition:
-     Call `sendFeedback(_:type:source:description:)` with the returned feedback to attach additional metadata to the feedback and send it.
-     */
-    public func createFeedback(screenshotOption: FeedbackScreenshotOption = .automatic) -> FeedbackEvent? {
+
+    func createFeedback(screenshotOption: FeedbackScreenshotOption = .automatic) -> FeedbackEvent? {
```

Comment:
It is because I left the same comments only in the public class `NavigationEventsManager` and I don't want to have duplicate docs in a non-public class.
This `NavigationCommonEventsManager` is a new non-public class that keeps an old Common Telemetry implementation. I decided to keep the git history for its methods from the original class.